scipy/spatial/transform/rotation.py

```diff
@@ -1,12 +1,71 @@
 from __future__ import division, print_function, absolute_import
 
 import numpy as np
+import scipy.linalg
+import warnings
 
 class Rotation(object):
     """Rotation in 3 dimensions.
 
     This class will include initializers from different representations, converters
     and some useful algorithms such as quaternion slerp and rotation estimation.
+
+    For initializing Rotations usage of `from_...` methods such as `from_quaternion`
+    is recommended instead of using `__init__`.
+
+    Methods
+    -------
+    from_quaternion
     """
-    def __init__(self):
-        pass
+    def __init__(self, quat, normalized=False):
+        self._single = False
+        # Try to convert to numpy array
+        quat = np.asarray(quat, dtype=float)
+
+        if quat.ndim not in [1, 2] or quat.shape[-1] != 4:
+            raise ValueError("`quat` should be of shape (4,) or (N x 4).")
```

Comment:
would be nice to add `, got shape %s' % (quat.shape,)`

Reply:
I support this. I prefer `.format(...)` though, @adbugger --- pick what you like more.